infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryServiceAdapter.java

```diff
@@ -1,30 +0,0 @@
-package org.corfudb.infrastructure.logreplication.infrastructure;
-
-import org.corfudb.infrastructure.logreplication.proto.LogReplicationClusterInfo;
-import org.corfudb.infrastructure.logreplication.proto.LogReplicationMetadata;
-
-import java.util.Map;
-import java.util.UUID;
-
-public interface CorfuReplicationDiscoveryServiceAdapter {
-
```

Comment:
Removing this interface would mean that the clusterManagerAdapter plugin implementation would have access to other methods in discoveryService as well.

Should we allow that?

Reply:
Totally valid point, we should bring it back and probably add this as part of the class description so its not lost.

Reply:
Done